### ISSUE TITLE
Add placeholder sonification

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#169]
IT would be pretty simple to add some placeholder sonification to balloons for rubbing, picking up charges and hitting a boundary.  This is described in many places throughout the design document, so we could get something in quickly just to get a feel for the flow of sound + description.

**Comment by terracoda** [comment]
@jessegreenberg, I would love this :-)

**Comment by jessegreenberg** [comment]
Done in the big commit above!  Kind of fun :) There are sound effects for the balloon picking up charges, rubbing on the sweater, and hitting a boundary in the play area.  It is possible to pick up charges *really* quickly, so I added a small delay so that you can hear individual beeps even if many charges are picked up at once.

Adding low priority label, since other descriptions take precedence, but this can be revisited any time.

Sonification hidden behind the 'sonification' query parameter.

**Comment by jessegreenberg** [comment]
We can close this issue, placeholder sonification has been added.  For more substantial work, a new issue should be opened.
